lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -219,60 +220,174 @@ sub overview {
 
 =over 4
 
-=item create()
+=item _eval_param_grouping()
 
-Creates a job given a list of settings passed as parameters. TEST setting/parameter
-is mandatory and should be the name of the test.
+Removes job-specific parameters from the specified hash and returns a new hash
+containing the removed parameters (where the keys are job suffixes).
+
+So "{foo => 1, bar:jobA => 2, baz:jobB => 3}" will be turned into "{foo => 1}"
+returnig "{jobA => {bar => 2}, jobB => {baz => 3}}".
```

Comment:
s/returnig/returning/, better yielding

Reply:
I'll change it to returning but yield is mainly used in the context of generators/co-routines so I wouldn't use it here.